windows/mdns_impl.c

```diff
@@ -2,13 +2,13 @@
 #include <stdio.h>
 
 
-#ifdef _WIN32
+
 #include <winsock2.h>
 #include <iphlpapi.h>
 #define sleep(x) Sleep(x * 1000)
-#else
-#include <netdb.h>
-#include <ifaddrs.h>
+
+#if (_DEBUG == 0)
```

Comment:
Could you please add instructions in the README.md under Windows section how can one enable this?

Reply:
Actually the flutter toolchain automatically sets this for debug builds.